apps/scraper-runner/src/main.ts

```diff
@@ -0,0 +1,79 @@
+import * as scholarsScrape from '@foodmedicine/scholars-scraper';
+import * as articleParser from '@foodmedicine/article-parser';
+import * as healthSiteScraper from '@foodmedicine/health-site-scraper';
+import * as fs from 'fs';
+import {
+  ParsedArticleParagraphStandalone,
+  ParsedArticle,
+  ParsedArticleParagraph,
+} from '@foodmedicine/interfaces';
+
+/**
+ * Get the top percentage of an array
+ * @param arr an array sorted in descending order
+ * @param percent percentage of items to be returned
+ */
+function getTopPercentage(arr: any[], percent = 5): any[] {
+  return arr.slice(0, Math.floor((arr.length * percent) / 100));
+}
+
+async function main() {
+  const healthRemedies = await healthSiteScraper.runAllScrapers();
+  const findCorrelatedParagraphsProms = healthRemedies.map(
+    async (healthRemedy) => {
+      const recommendationResults = healthRemedy.recommendations.map(
+        (recommendation) =>
+          findCorrelatedParagraphs(healthRemedy.impacted, recommendation)
+      );
+      return await Promise.all(recommendationResults);
+    }
+  );
+  await Promise.all(findCorrelatedParagraphsProms);
+}
+
+async function findCorrelatedParagraphs(
+  impacted: string,
+  recommendation: string
+) {
+  console.log(
+    `Starting to find correlation of ${recommendation} for ${impacted}`
+  );
+  const articleHeads = await scholarsScrape.runScholarsScraper(
+    impacted,
+    recommendation
+  );
+  const downloadProms: Promise<ParsedArticle>[] = articleHeads.map(
```

Comment:
Nit: prefer non-abbreviated variable names if possible to help with readability down the road